tests/puppeteer/pages/BO/BO_dashboard.js

```diff
@@ -0,0 +1,13 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class BO_DASHBOARD extends CommonPage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'PrestaShop';
```

Comment:
Please use `Dashboard *` instead and remove it from BO_commonPage.js

Reply:
Done